src/argilla/tasks/datasets/push.py

```diff
@@ -69,3 +53,7 @@ def push_to_huggingface(
     except Exception as e:
         typer.echo("An unexpected error occurred when trying to push the `FeedbackDataset` to the HuggingFace Hub")
         raise typer.Exit(code=1) from e
+
+    console.print(
+        f":sparkles: `FeedbackDataset` successfully pushed to the :hugs: HuggingFace Hub at https://huggingface.co/{repo_id}"
```

Comment:
Should we keep the `:sparkles:` emoji here? Maybe we can just remove it, as we're not using emojis along the CLI so maybe this is a bit off